### PR TITLE
readme: Update the instructions

### PR DESCRIPTION
**Pull request by iammarco11**
I have updated the instructions with points and links required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,16 @@ $ source <virtualenv name>/bin/activate
 $ pip install -r requirements.txt
 ```
 
+* Make the directory for the log storage
+```
+$ mkdir /var/log/nawab
```
Comment (Aniketh01) [inline README.md]: Need sudo

File: README.md [hunk 2]
```diff
@@ -39,6 +39,16 @@ $ source <virtualenv name>/bin/activate
 $ pip install -r requirements.txt
 ```
```
Comment (Aniketh01) [inline README.md:44]: In this section we have discussed about how to reproduce the project. 

Above all these, we should explain how to use it. That is, where to find the nawab twitter and telegram instance and how to get it started

**Review by Aniketh01 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (iammarco11) [reply README.md:44]: We can make that a separate section after the initial version is released?

File: README.md (hunk 1 above)
Comment (iammarco11) [reply README.md]: ok

**Review by Aniketh01 — APPROVED** [review]
LGTM!

**Pull request merged**